CHT/CHT.C

```diff
@@ -85,86 +85,26 @@ std::string preciceAdapter::CHT::ConjugateHeatTransfer::determineSolverType()
     // NOTE: When coupling a different variable, you may want to
     // add more cases here. Or you may provide the solverType in the config.
 
-    std::string solverType;
-
-    // Determine the solver type: Compressible, Incompressible or Basic.
-    // Look for the files transportProperties, turbulenceProperties,
-    // and thermophysicalProperties
-    bool transportPropertiesExists = false;
-    bool turbulencePropertiesExists = false;
-    bool thermophysicalPropertiesExists = false;
+    std::string solverType = "basic";
 
-    if (mesh_.foundObject<IOdictionary>("transportProperties"))
-    {
-        transportPropertiesExists = true;
-        DEBUG(adapterInfo("Found the transportProperties dictionary."));
-    }
-    else
-    {
-        DEBUG(adapterInfo("Did not find the transportProperties dictionary."));
-    }
+    dimensionSet compressible_pressure_dimension(1, -1, -2, 0, 0, 0, 0);
```

Comment:
Why `snake_case` here? I would prefer something like `pressureDimensionsCompressible` and `pressureDimensionsIncompressible`.

Reply:
Yes, of course!